overturemaps/cli.py

```diff
@@ -99,11 +99,19 @@ def cli():
     type=click.Choice(get_all_overture_types()),
     required=True,
 )
-def download(bbox, output_format, output, type_):
+@click.option(
+    "-r",
+    "--release",
+    "release_",
+    default='2025-06-25.0',
+    type=click.Choice(get_all_overture_releases()),
+    required=False,
+)
+def download(bbox, output_format, output, type_, release_):
     if output is None:
         output = sys.stdout
 
-    reader = record_batch_reader(type_, bbox)
+    reader = record_batch_reader(type_, bbox, release_)
```

Comment:
Ensure release is in the list from get_all_releases and output a reasonable error if not.